docs/resources/aviatrix_edge_neo.md

```diff
@@ -0,0 +1,126 @@
+---
+subcategory: "Multi-Cloud Transit"
+layout: "aviatrix"
+page_title: "Aviatrix: aviatrix_edge_neo"
+description: |-
+  Creates Aviatrix Edge NEO
+---
+
+# aviatrix_edge_neo
+
+The **aviatrix_edge_neo** resource creates the Aviatrix Edge NEO.
+
+## Example Usage
+
+```hcl
+# Create an Edge NEO
+resource "aviatrix_edge_neo" "test" {
+  account_name               = "edge_neo-account"
+  gw_name                    = "neo-test"
+  site_id                    = "site-123"
+  device_id                  = "abcd123"
+  gw_size                    = "small"
+  management_interface_names = ["eth2"]
+  lan_interface_names        = ["eth1"]
+  wan_interface_names        = ["eth0"]
+
+  interfaces {
+    name          = "eth0"
+    type          = "WAN"
+    ip_address    = "10.230.5.32/24"
+    gateway_ip    = "10.230.5.100"
+    wan_public_ip = "64.71.24.221"
+  }
+
+  interfaces {
+    name       = "eth1"
+    type       = "LAN"
+    ip_address = "10.230.3.32/24"
+  }
+
+  interfaces {
+    name        = "eth2"
+    type        = "MANAGEMENT"
+    enable_dhcp = false
+    ip_address  = "172.16.15.162/20"
+    gateway_ip  = "172.16.0.1"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+### Required
+* `account_name` - (Required) Edge NEO account name.
+* `gw_name` - (Required) Edge NEO name.
+* `site_id` - (Required) Site ID.
+* `device_id` - (Required) Device ID.
+* `gw_size` - (Required) Gateway size. Valid values: "small", "medium", "large", "x-large".
+* `wan_interface_names` - (Required) List of WAN interface names. Default value: ["eth0"].
+* `lan_interface_names` - (Required) List of LAN interface names. Default value: ["eth1"].
+* `management_interface_names` - (Required) List of management interface names. Default value: ["eth2"].
+* `interfaces` - (Required) WAN/LAN/MANAGEMENT interfaces.
+    * `name` - (Required) Interface name.
+    * `type` - (Required) Type. Valid values: WAN, LAN, or MANAGEMENT.
+    * `bandwidth` - (Optional) Bandwidth.
```

Comment:
what is bandwidth? What type of values are accepted? Description isnt clear, esp since its not shown in the example code either